src/main/java/org/hyperskill/hstest/stage/StageTest.java

```diff
@@ -138,6 +139,8 @@ private void printTestNum(int num) {
 
     @Test
     public final void start() {
+        CheckLibraryVersion checkLibraryVersion = new CheckLibraryVersion();
+        checkLibraryVersion.checkVersion();
```

Comment:
Every executable code of `hs-test` should be under `try-catch` block. Including this one. It guarantees that in case something goes wrong, the user would still get some message from testing (yes, it would be `Unexpected error` but it is better than just crash with exception)

Reply:
We have two variants to check, 
`
if (!currentVersion.equals(latestVersion)) {
            if (currentVersion.split("\\.")[0] != latestVersion.split("\\.")[0]) {
                throw new RuntimeException(getFeedback());
            } else {
                isLatestVersion = false;
            }
        }
` 
`
if (currentVersion.split("\\.")[0] != latestVersion.split("\\.")[0]) {
                throw new RuntimeException(getFeedback());
`
if the user's version and Git do not equal in main index X.0.0 user has RuntimeException(getFeedback()); and process stoped,
and light checking when the user has failed on the test case.

Reply:
What kind of feature should it be for the release of `(X+1).0.0` so that every user needs to update to it right now? Every major release added support for major technology or language. For example: support of Go testing, support of Bash testing, support of Spring Boot testing. In case the users already testing their applicarion chances are that they simply don't need new major language or technology supported in `hs-test`. Because whatever they are testing already supported in `hs-test`.